js/client-battle-tooltips.js

```diff
@@ -410,7 +410,13 @@ var BattleTooltips = (function () {
 			var exacthp = '';
 			if (pokemonData) exacthp = ' (' + pokemonData.hp + '/' + pokemonData.maxhp + ')';
 			else if (pokemon.maxhp == 48) exacthp = ' <small>(' + pokemon.hp + '/' + pokemon.maxhp + ' pixels)</small>';
-			text += '<p>HP: ' + pokemon.hpDisplay() + exacthp + (pokemon.status ? ' <span class="status ' + pokemon.status + '">' + pokemon.status.toUpperCase() + '</span>' : '') + '</p>';
+			text += '<p>HP: ' + pokemon.hpDisplay() + exacthp + (pokemon.status ? ' <span class="status ' + pokemon.status + '">' + pokemon.status.toUpperCase() + '</span>' : '');
+			if (pokemon.status === 'tox') {
+				text += ' Next damage: ' + Math.floor(100 / 16) * Math.min(pokemon.statusData.toxicTurns, 15) + '%';
+			} else if (pokemon.status === 'slp') {
+				text += ' Turns asleep: ' + pokemon.statusData.sleepTurns;
+			}
```

Comment:
Don't show these in hardcore mode.

Reply:
It doesn't, there's a check for hardcoreMode above this that makes hardcoreMode never enter this section. https://github.com/Zarel/Pokemon-Showdown-Client/blob/master/js/client-battle-tooltips.js#L408

Reply:
Oh okay.